### DISCUSSION TITLE
Automatic login/auth

### DISCUSSION THREAD

**Discussion by thejoeosborne** [mxrch/GHunt#490] [Q&A]
Hi! Has anyone had any luck setting up the auth flow to run automatically? For example, I'm hoping to use ghunt in a Docker container, the only problem is I currently need to login to google to generate the oauth_token to generate the master token. Any ideas on automating the auth so I don't have to manually login to google? I've tried writing a script to login using Puppeteer but it doesn't work with a headless browser, the google login doesn't trust it

**Comment by zxelzy** [comment]
I don't recommend automating Google logins because their anti-bot detection system is quite strict and can lead to account restrictions. Alternatively, you can manually authenticate once, then store the resulting token and inject it into the container via a volume or environment variable.
